packages/server/lib/environment.coffee

```diff
@@ -19,6 +19,8 @@ try
   app = require("electron").app
   app.commandLine.appendSwitch("disable-renderer-backgrounding", true)
   app.commandLine.appendSwitch("ignore-certificate-errors", true)
+  ## https://github.com/cypress-io/cypress/issues/2376
+  # app.commandLine.appendSwitch("autoplay-policy", "no-user-gesture-required")
```

Comment:
why is this commented out? won't this fail in electron? this is super confusing

Reply:
sorry, yeah, I had issues merging and I think everything just got a bit hectic in the commits.